srcpkgs/consolation/template

```diff
@@ -0,0 +1,17 @@
+# Template file for ''consolation''
+pkgname='consolation'
+version='0.0.8'
+revision='1'
+build_style='gnu-configure'
+hostmakedepends='pkg-config>=0.9 libinput-devel>=1.15'
```

Comment:
`libinput-devel` belongs in `makedepends`. Also remove the version patterns, there is always just one version for build time dependencies and the version has to match the version of the template so its not required.